riggerlib/client.py

```diff
@@ -115,6 +115,7 @@ def terminate(self):
         try:
             self._request({'event_name': 'shutdown'})
             self.ready = False
+            self.zmq.close()
```

Comment:
we occasionally have a logging call from one of the keepalive threads for the browser/sprout
those thread calls where identified as most probable reason for the jenkins master hangups